apps/dev-recruiters/src/components/modules/FilterPage/RolesFilterComponent/RolesFilterList/SearchRoles/index.tsx

```diff
@@ -57,12 +75,24 @@ function SearchRole({ selectedRoleLabel, opportunities }: Props) {
               Oops, there are currently no matching roles. Check out other
               suggested roles below!
             </EmptyRolesContainer>
-            <OpenRolesText>Suggested roles(1)</OpenRolesText>
-            <SuggestedRole />
+            <OpenRolesText>
+              Suggested roles({suggestedRole.length})
+            </OpenRolesText>
+            <RolesContainer>
+              {suggestedRole?.map((role, index) => {
+                return (
+                  <RoleCard
+                    key22={index}

```

Comment:
Just for my understanding, why is it call `key22`?

Reply:
Well I don't know. Its there in RoleCard component from before. And we have been using it as it is.

Reply:
Should I change key22 variable Name everywhere?

Reply:
We can rename it in the future.

Reply:
Sure